index.html

```diff
@@ -18,18 +31,18 @@
     <![endif]-->
 
   </head>
-  <body>
+  <body >
 
     <div class="container">
       <div class="pycon-logo text-center">
-        <img src="pycon_india_2017.png" style="max-width: 75%;">
+        <img src="pycon_india_2017.png" style="max-width: 75%;"></img>
       </div>
       <div class="imp-details text-center">
         <h2>Venue: IIM Lucknow (Noida Campus)</h2>
         <h2>Dev Sprints: 2 - 3 November</h2>
         <h2>Main Conference: 4 - 5 November</h2>
       </div>
-
+<div class="BOX">
```

Comment:
prefer naming css classes in small alphabets separated by hyphens in case it is more than one word.